webui/ocw/lib/db.py

```diff
@@ -54,47 +55,53 @@ def __update_run():
     from . import EC2db
     from . import azure
     from . import gce
+    global update_error
```

Comment:
don't like the idea of this variable . it keeping info about error only till next update round. this give a room for different messy scenarios . Like one time error , sporadic error , different errors with each run. I would drop it at all and setup mailing of errors . it would be good enough and is_updating function should just answer if update is currently running or not. Probably would be nice also add time of last successful update

Reply:
also if you drop it  this will ease implementation of my idea to moving try/catch into providers functions :)

Reply:
Ok, I will send a email notification when an error occur. But also display something in  the GUI. E.g if sending mail fail, at least in the GUI is something visible.

Reply:
only in case you are lucky to check UI before next iteration :) what about my idea to just display time of last successful update ? if it is too old you will understand that something is wrong and go to check the logs to see what is actually wrong

Reply:
done